### DISCUSSION TITLE
Stack Tailwind

### DISCUSSION THREAD

**Discussion by ramon316** [awais-vteams/laravel-crud-generator#29] [Q&A] [answered]
![Captura de pantalla 2024-06-04 115721](https://github.com/awais-vteams/laravel-crud-generator/assets/15043691/66d604a1-803f-4a48-a4dd-d00de25a7ef5)
Hi, When i run "php artisan make:crud banks" by default use bootstrap, but when i use "php artisan make:crud tailwind" this command does not work, i don't understand, can anyone help me ?

**Comment by johnsmith-fcs** [comment, marked as answer]
@ramon316 Which Laravel version do you have? when you write `php artisan make:crud` they'll ask you which stack you want to choose like the below image 
![image](https://github.com/awais-vteams/laravel-crud-generator/assets/121596044/ccddc1f7-ed69-42b5-9a64-170a7ce4158a)

Also in your comment, I am missing the table name. It should be like `php artisan make:crud banks tailwind `
